lib/streamlit/runtime/scriptrunner/magic.py

```diff
@@ -110,16 +117,23 @@ def _insert_import_statement(tree):
 
 
 def _build_st_import_statement():
-    """Build AST node for `import streamlit as __streamlit__`."""
-    return ast.Import(names=[ast.alias(name="streamlit", asname="__streamlit__")])
+    """Build AST node for `import magic_funcs as __streamlitmagic__`."""
+    return ast.Import(
+        names=[
+            ast.alias(
+                name="streamlit.runtime.scriptrunner.magic_funcs",
+                asname=MAGIC_MODULE_NAME,
+            )
+        ]
+    )
 
 
 def _build_st_write_call(nodes):
-    """Build AST node for `__streamlit__._transparent_write(*nodes)`."""
+    """Build AST node for `__streamlitmagic__.transparent_write(*nodes)`."""
     return ast.Call(
         func=ast.Attribute(
-            attr="_transparent_write",
-            value=ast.Name(id="__streamlit__", ctx=ast.Load()),
+            attr="transparent_write",
```

Comment:
It's now `transparent_write` instead of `_transparent_write` (because it no longer lives in the top-level streamlit import).